src/ui/ResponsiveComponents/ResponsiveDropdown/ResponsiveDropdownHeader.ts

```diff
@@ -1,5 +1,10 @@
 import { Dom, $$ } from '../../../utils/Dom';
 
+export enum ResponsiveDropdownHeaderEvent {
+  OPEN = 'responsiveDropdownHeaderOpen',
```

Comment:
Hmm, I didn't know using `=` sign worked. I thought you should use `:`?

Reply:
It's a string enum. That's the syntax.

I could write it
```
export enum ResponsiveDropdownHeaderEvent {
  OPEN,
  CLOSE
}
```

Which would make it a standard number enum, where `ResponsiveDropdownHeaderEvent.OPEN` is `0` and `.CLOSE` is 1.

But in this case I need a string enum, to trigger events